src/pages/ProductList/ItemDisplay/ItemDisplay.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { FiHeart } from 'react-icons/fi';
+import { FaHeart } from 'react-icons/fa';
+import './ItemDisplay.scss';
+
+function ItemDisplay({ data }) {
+  const [heartChange, setHeartChange] = useState(false);
+
+  function wishListHandle() {
+    setHeartChange(!heartChange);
+  }
```

Comment:
영운님! 함수형 업데이트라는 키워드를 한 번 검색해보시고, 적용하면 좋을 듯 합니다.
함수형 업데이트를 적용하게 되면 아래와 같이 사용하게 되실겁니다!
```suggestion
  function wishListHandle() {
    setHeartChange(heartChange => !heartChange);
  }
```

Reply:
수정 사항 반영했습니다.